client/src/tests/procedure_tests.rs

```diff
@@ -398,24 +398,26 @@ async fn usecase_secp256k1() -> Result<(), Box<dyn std::error::Error>> {
         };
         let evm_addr = client.execute_procedure(evm_address).unwrap();
 
-        assert_eq!(&evm_addr, pk.to_evm_address().as_ref());
+        assert_eq!(&evm_addr, pk.evm_address().as_ref());
 
         let msg = fresh::variable_bytestring(4096);
 
         let secp256k1_ecdsa_sign = Secp256k1EcdsaSign {
+            flavor: Secp256k1EcdsaFlavor::Secp256k1EcdsaKeccak256,
             private_key: sk,
             msg: msg.clone(),
         };
-        let mut sig_bytes: [u8; secp256k1_ecdsa::Signature::LENGTH] =
+        let mut sig_bytes: [u8; secp256k1_ecdsa::RecoverableSignature::LENGTH] =
             client.execute_procedure(secp256k1_ecdsa_sign).unwrap();
 
-        let sig = secp256k1_ecdsa::Signature::try_from_bytes(&sig_bytes).unwrap();
-        assert!(pk.verify(&sig, &msg));
-        assert_eq!(pk, sig.verify_recover(&msg).unwrap());
+        let sig = secp256k1_ecdsa::RecoverableSignature::try_from_bytes(&sig_bytes).unwrap();
+        // assert!(pk.verify(&sig, &msg));
```

Comment:
?

Reply:
Fixed. Forgot that we have `impl AsRef<Signature> for RecoverableSignature`.